l10n_it_ricevute_bancarie/models/partner/partner.py

```diff
@@ -20,3 +20,11 @@ class ResPartner(models.Model):
     riba_exclude_expenses = fields.Boolean(
         string="Exclude expenses Ri.Ba."
     )
+    riba_policy_expenses = fields.Selection(
+        [
+            ('one_a_month', 'More invoices, one expense per Month'),
+            ('unlimited', 'One expense per maturity')
```

Comment:
Non so se è più chiaro, se si emettono 2 riba per la stessa scadenza ci sono 2 addebiti, cmq va bene uguale.
```suggestion
            ('one_a_month', 'More invoices, one expense per Month'),
            ('unlimited', 'One expense per issued maturity')
```